modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -32,7 +32,7 @@ To successfully install ROSA clusters, use the latest version of the ROSA CLI (`
 //REGION="<region>"
 //----
 
-. You can create your {hcp} for ROSA cluster with one of the following commands. 
+. You can create your {hcp-title} cluster with one of the following commands. 
```

Comment:
Lines 42/50: The word "ingress" is capitalized here, but it isn't usually capitalized in our documentation.